.changeset/pre.json

```diff
@@ -12,6 +12,7 @@
     "@solana/codecs": "2.0.0-preview.1",
     "@solana/compat": "2.0.0-preview.1",
     "@solana/errors": "2.0.0-preview.1",
+    "@solana/fast-stable-stringify": "1.0.0",
```

Comment:
I don't think this needs to be added here in this PR. This comes after the first one is published, I believe.